lib/shelley/exe/local-cluster.hs

```diff
@@ -201,6 +202,9 @@ import qualified Data.Text as T
 --     "virtual hard forks" in the node config files.
 --     The final era can be changed with this variable.
 --
+-- - TOKEN_METADATA_SERVER  (default: none)
+--     Use this URL for the token metadata server.
```

Comment:
👍 

Not important now but:
- _Maybe_ worth including in the integration tests as well. IIRC all the other ENV vars are the same.
- One idea could be to allow file://path/to/000000000000.json as a special case, which would cause the wallet to simply read the json instead of sending a POST request. Maybe could be useful for testing locally.

Reply:
1. Yes good idea - we can sort it later.
2. Also a good idea, but perhaps a better idea would be to see how possible it is to run the _real_ metadata-server with the test json file.